src/lib/util/tests-comparison.ts

```diff
@@ -0,0 +1,13 @@
+
+export function isPrimitivesArraysMatchAtAnyOrder<T = any>(actualArray: T[], expectedArray: T[]): void {
```

Comment:
I kind of like this method for assertion purposes. I just have some change requests:
- the method requires some comments, that describe what it does. These comments should not be located inside the method but before it. They should also include the an explanation of the input parameters
- the method requires a better name which also describes what it does. Ideally it should start with `assert`.
- I would like if the method could also check the order of the elements inside the arrays, but I'm not sure if this makes sense.

Reply:
> I kind of like this method for assertion purposes. I just have some change requests:
> 
> * the method requires some comments, that describe what it does. These comments should not be located inside the method but before it. They should also include the an explanation of the input parameters
> * the method requires a better name which also describes what it does. Ideally it should start with `assert`.
> * I would like if the method could also check the order of the elements inside the arrays, but I'm not sure if this makes sense.

About the first point:
**comments needed when the code isn't self-explanatory**, I tried to avoid comments because comments needed to be maintained.

## Second point comment
I will change the function name as you suggested

## Third point comment
I can but I think it should be in different function

Reply:
> I kind of like this method for assertion purposes. I just have some change requests:
> 
> * the method requires some comments, that describe what it does. These comments should not be located inside the method but before it. They should also include the an explanation of the input parameters
> * the method requires a better name which also describes what it does. Ideally it should start with `assert`.
> * I would like if the method could also check the order of the elements inside the arrays, but I'm not sure if this makes sense.

### About the first point:
**comments needed when the code isn't self-explanatory**, I tried to avoid comments because comments needed to be maintained.

## Second point comment
I will change the function name as you suggested

## Third point comment
I can but I think it should be in different function